apps/web/src/components/RightSideBarActivities.tsx

```diff
@@ -21,6 +24,8 @@ export const RightSideBarActivites: React.FC = () => {
       setItems(newCareActivity);
       //Set checked ids to previously selected ids when care activity changes
       values.careActivities = _.filter(values.careActivityBundle[values.careActivityID]);
+      //Empty array when careActivityID changes
+      values.checked = [];
```

Comment:
Have already added some comments on the Tags PR regarding this

Reply:
Removed and updated to develop version.